tests/acceptance/features/steps/keystone_token.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-WARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+# -*- coding: utf-8 -*-
+import requests
+from behave import when, then, given
+from skuld.expired_users import ExpiredUsers
+from tests.acceptance.commons.configuration import KEYSTONE_URL, TOKEN_LENGTH
+
+__author__ = 'fla'
+
+
+MULTIPLE_CHOICE = 300
+
+
+@given(u'a valid tenantName, username and password')
+def step_impl(context):
```

Comment:
I can see a lot of methods named with the same name _step_impl_. I prefer using different ones for each function.